locales/de.json

```diff
@@ -0,0 +1,22 @@
+{
+  "settings": {
+    "intro": "Für jede hinzugefügte Variable werden 5 Tags erstellt: <ul><li>{variable} Laufzeit</li><li>{variable} Differenz</li><li>{variable} Währung</li><li>{variable} Berechnung</li><li>{variable} Dezimalstellen</li></ul> Diese werden von den verschiedenen Flow Karten gefüllt und können dann in den Flows verwendet werden.",
```

Comment:
The values set below on line 16 till 20 will be used in the variable name so I think to have correct capitalisation it would be:
`"intro": "Für jede hinzugefügte Variable werden 5 Tags erstellt: <ul><li>{variable} laufzeit</li><li>{variable} differenz</li><li>{variable} währung</li><li>{variable} berechnung</li><li>{variable} dezimalstellen</li></ul> Diese werden von den verschiedenen Flow Karten gefüllt und können dann in den Flows verwendet werden.",`